csm/conf/setup.py

```diff
@@ -460,6 +460,17 @@ def _logrotate(self):
         else:
             raise CsmSetupError("logrotate failed. %s dir missing." %const.LOGROTATE_DIR)
 
+    def _set_fqdn_for_nodeid(self):
+        nodes = Setup.get_salt_data(const.PILLAR_GET, const.NODE_LIST_KEY)
+        if nodes:
+            for each_node in nodes:
```

Comment:
We should log if some values are missing

Reply:
Comment Resolved Please review